drivers/usb/device/usb_dc_stm32.c

```diff
@@ -173,9 +181,15 @@ static int usb_dc_stm32_clock_enable(void)
 	struct device *clk = device_get_binding(STM32_CLOCK_CONTROL_NAME);
 	struct stm32_pclken pclken = {
 
+#ifdef CONFIG_USB_HS_BASE_ADDRESS
+		.bus = STM32_CLOCK_BUS_AHB1,
+		.enr = LL_AHB1_GRP1_PERIPH_OTGHS
+#else /* CONFIG_USB_HS_BASE_ADDRESS */
+
 #ifdef USB
```

Comment:
In which case will the macro 'USB' be enabled?

Reply:
STM32Cube HAL defines USB on SoCs with USB device peripheral (not OTG FS or OTG HS) e.g. STM32103RB (see section 23 in RM0008).